### PR TITLE
added readme

### PR DESCRIPTION
**Pull request by DanyNest**
added description to readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,47 @@
...
+With that app you can track what movie sessions are available, make and complete orders(buy tickets, adding to your shopping cats) and so on. 
+
+
+![cinema pict](https://mmr.ua/wp-content/uploads/2021/05/mgm-studios-1200x580.jpg)
+____
+
+###  <span style="color: RED">How</span><span style="color: black"> to</span><span style="color: orange"> run: </span>
+1. First you'll need to connect to your database(MySQL) using db.properties file in resources package.
+2. Then configure [Tomcat](https://tomcat.apache.org/) local server. 
+3. There is implemented DataInitializer class, so you already have login `admin@123` and password `admin123`, so you don't need to be registered throw POSTMAN to have access to
```
Comment (KseniiaMakarova) [inline README.md]:
```suggestion
3. There is implemented DataInitializer class, so you already have login `admin@123` and password `admin123`, so you don't need to be registered through POSTMAN to have access to
```

File: README.md [hunk 2]
```diff
@@ -1 +1,47 @@
...
+the service. Some services(endpoints) are allowed only for those who has role ADMIN, and some for those who is User, in our case we'll have both.
+4. To test all the links and services, you should use download [POSTMAN](https://www.postman.com/), using it, is much easier to test the service.
+
+___
+#### <span style="color:ORANGE">Endpoints</span>
+1. POST: `/register` `/logout` - allowed for everyone.
```
Comment (KseniiaMakarova) [inline README.md]: logout?

File: README.md [hunk 3]
```diff
@@ -1 +1,47 @@
...
+2. POST: `/orders/complete` - allowed for role `USER`.
+3. POST: `/cinema-halls, /movies, /movie-sessions`- allowed for role `ADMIN`.
+4. PUT: `/shopping-carts/movie-sessions` - allowed for role `USER`.
+5. PUT: `/movie-sessions/**` - allowed for role `ADMIN`.
+6. DELETE: `/movie-sessions/**` - allowed for role `ADMIN`.
+7. GET: `/shopping-carts/by-user`, `/orders` - allowed for role `USER`.
+8. GET:`/users/by-email` - allowed for role `ADMIN`.
+9. GET: `/cinema-halls`, `/movies`, `/movie-sessions/available`, `/movie-sessions/**`- allowed for roles `ADMIN` and `USER`.
```
Comment (KseniiaMakarova) [inline README.md]:
```suggestion
9. GET: `/cinema-halls`, `/movies`, `/movie-sessions/available`- allowed for roles `ADMIN` and `USER`.
```

**Review by KseniiaMakarova — COMMENTED** [review]
- https://github.com/DanyNest/cinema-app/blob/master/src/main/java/cinema/config/WebConfig.java -> limit to `controller` folder
- check all your models, add annotations: @Table, @Column, @JoinTable, @JoinColumn - there should not be any camelCase in your db

File: README.md (hunk 1 above)
Comment (DanyNest) [reply README.md]: T9

File: src/main/java/cinema/config/AppConfig.java [hunk 4]
```diff
@@ -14,7 +14,7 @@
 
 @Configuration
 @PropertySource("classpath:db.properties")
-@ComponentScan(basePackages = {"cinema"})
+@ComponentScan(basePackages = {"cinema.controller"})
```
Comment (KseniiaMakarova) [inline src/main/java/cinema/config/AppConfig.java]: why here, and not in WebConfig?

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**